src/coreclr/jit/assertionprop.cpp

```diff
@@ -777,10 +777,14 @@ void Compiler::optPrintAssertion(AssertionDsc* curAssertion, AssertionIndex asse
 
     if (curAssertion->op1.kind == O1K_LCLVAR)
     {
-        printf("V%02u", curAssertion->op1.lcl.lclNum);
-        if (curAssertion->op1.lcl.ssaNum != SsaConfig::RESERVED_SSA_NUM)
+        if (optLocalAssertionProp)
```

Comment:
Is this logic reversed? If we have local AP we want the var.

Reply:
Oops! Luckily, this is JitDump only